### PR TITLE
Add place schema

### PR DESCRIPTION
**Pull request by cbaines**
https://trello.com/c/t3DXuS2t/660-migrate-place-format-to-rendered

### CODE REVIEW

File: formats/place/frontend/examples/find-regional-passport-office.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    ]
+  },
+  "description": "Find a Passport Customer Service Centre for Fast Track and Premium service passport applications - offices are in Belfast, Durham, Glasgow, Liverpoool, London, Newport and Peterborough",
+  "details": {
+    "external_related_links": [
+
+    ],
+    "introduction": "<p>Enter your postcode to find your nearest Passport Customer Service Centre.</p>\n\n<p>You must <a href=\"/get-a-passport-urgently\">book an appointment</a> for an urgent passport application before you call or visit a centre. Use the <a href=\"/apply-renew-passport\">normal service</a> if you don’t need a passport urgently.</p>\n\n<p>You’ll need to bring proof of your identity.</p>\n",
+    "more_information": "<p>There are Passport Customer Service Centres in:</p>\n\n<ul>\n  <li>Belfast</li>\n  <li>Durham</li>\n  <li>Glasgow</li>\n  <li>Liverpool</li>\n  <li>London</li>\n  <li>Newport</li>\n  <li>Peterborough</li>\n</ul>\n",
+    "need_to_know": "\n",
```
Comment (fofr) [inline formats/place/frontend/examples/find-regional-passport-office.json]: Should this be included?

**Comment by brenetic** [comment]
@fofr  We have fixed a bug in Publisher so it will no longer send fields which are nil or empty. Updated the example accordingly.

**Comment by whoojemaflip** [comment]
Other mainstream formats come with a `publisher_v2/example` document, for validating the output of publisher. Any plans to include one here?

**Comment by cbaines** [comment]
> Other mainstream formats come with a publisher_v2/example document, for validating the output of publisher. Any plans to include one here?

Added now :)

**Pull request merged**